generator.yaml

```diff
@@ -290,6 +290,90 @@ resources:
         from:
           operation: DescribeHyperParameterTuningJob
           path: OverallBestTrainingJob
+  DataQualityJobDefinition:
+    exceptions:
+      errors:
+          404:
+            code: ResourceNotFound
+      terminal_codes:
+        - ResourceLimitExceeded
```

Comment:
is there any ambiguity here on which error codes we add here - as in does it vary per resource ? I was away so missed most of these conversations, if any.